packages/bsky/proto/bsky.proto

```diff
@@ -1095,7 +1095,19 @@ message GetFollowsFollowingResponse {
 message PingRequest {}
 message PingResponse {}
 
+// Subscription entitlement
+message SubscriptionEntitlement {
+  string did = 1;
```

Comment:
I think I'd remove the `did` here and just rely on ordering of arrays as we do in some other examples (take `GetDidsByHandles` for example which does not return the handle in the response)